modules/targetAcquisition/taxi/TaxiWorker.py

```diff
@@ -9,9 +9,8 @@ def taxi_worker(pause, exitRequest, pipelineIn, pipelineOut):
         pause.release()
         
         frame = pipelineIn.get()
-        taxi.main(frame)
-
-        pipelineOut.put()
+        if taxi.main(frame).latestDistance == 0:
+            pipelineOut.put(taxi.main(frame))
```

Comment:
Minor thing, don't call main over and over again. Call taxi.main(frame) and save it to a variable like ```command = taxi.main(frame)``` use that variable instead